R/array_api.R

```diff
@@ -1653,7 +1653,7 @@ ArrayApi <- R6::R6Class(
                                  body = body,
                                  ...)
       # MANUAL EDIT AFTER OPENAPI AUTOGEN
-      wrap_as_api_response(resp)
+      .wrap_as_api_response(resp)
```

Comment:
Just as an FYI, I like this old idiom of marking internal functions with a leading dot (with would then exclude them from the common regular expression in `NAMESPACE`).  These days exports are more explicit via the roxygen tags and the _generated_ `NAMESPACE` (as opposed to manual edits and/or the regexp) so it is less clear what is / isn't exported.  The dot helps.

That said, you can maybe also govern via OpenAPI which functions do / do not get an `@export` tag.

Reply:
Thanks @eddelbuettel ! The context behind the leading dot (not yet included in this unpublished draft) is the docsite at https://tiledb-inc.github.io/TileDB-Cloud-R/index.html -- using our current setup, these internal-only functions _are_ included in the listing (even though there's no `@export` for them as of this PR) so I am using the leading-dot convention, along with a `matches` in `_pkgdown.yml`, so I can split these out as separate.

WIP/localhost doc pages look like this:
<img width="1403" alt="Screen Shot 2021-12-07 at 4 32 13 PM" src="https://user-images.githubusercontent.com/2008794/145109347-c91cfedd-b11e-41a3-bc85-3436a2ba9276.png">
<img width="1240" alt="Screen Shot 2021-12-07 at 4 32 20 PM" src="https://user-images.githubusercontent.com/2008794/145109360-cd5092d0-01b1-4fd6-bfc7-341d705f4280.png">

Reply:
I have fairly little `pkgdown`-fu but I would be surprised if one couldn't hide internal functions here too.  Having the multiple layers of internal (possibly undocumented), internal (possibly documented), exported has been pretty consistent and standard with R.